dist/berlin-transport-card.js

```diff
@@ -134,6 +134,8 @@ class BerlinTransportCard extends HTMLElement {
             .delay {
                line-height: 2em;
                font-size: 70%;
+               text-align: right;
+               min-width: 1.5em;
```

Comment:
Should this be rather
```suggestion
               min-width: 2ch;
```